yarn.lock

```diff
@@ -4067,6 +4093,20 @@ gitconfiglocal@^1.0.0:
   dependencies:
     ini "^1.3.2"
 
+gitlab@^10.0.0:
+  version "10.0.0"
+  resolved "https://registry.yarnpkg.com/gitlab/-/gitlab-10.0.0.tgz#71dec3b79a7716263f7c63bb465b66e76da6348b"
+  integrity sha512-I8teXj+JzYokyPsoMWRbDKqesNafwRlOqrq7tuNIaeotvsVr+jb9WzEMen+Zy/dHLI6aHN/fp4N5pe3plrdJjQ==
+  dependencies:
+    form-data "^2.3.3"
+    humps "^2.0.1"
+    ky "^0.11.0"
+    ky-universal "^0.2.1"
+    li "^1.3.0"
+    query-string "^6.8.1"
+    randomstring "^1.1.5"
+    universal-url "^2.0.0"
```

Comment:
I'm worried that we have both gitlab 10, as we have gitlab locked to v6  in the app - have you tried updating gitlab to v10 and seeing if that can fix it in general?

Reply:
It might be worth giving this another shot now that #903 is in

Reply:
> At first, I've tried to use just node script without dangerjs with last `gitlab` as dependency. This problem was reproduced.
> 
> Gitlab have an option to fix it - not very handy but working - send custom `requester` as parameter. I made the fix with extra options actually, this function in my code fixes my problem:
> 
> ```
> function buildAgent() {
>     const { GITLAB_SSL_KEY, GITLAB_SSL_CERT } = process.env;
> 
>     if (GITLAB_SSL_KEY || GITLAB_SSL_CERT) {
>         const key = readKey(GITLAB_SSL_KEY);
>         const cert = readKey(GITLAB_SSL_CERT);
> 
>         return new https.Agent({
>             key,
>             cert,
>         })
>     }
>     return;
> }
> ```
> 
> If it would be better for this PR, I can downgrade some dependencies in my project.
> 
> Or, maybe, would be better to extend Dangerjs API to allow sending extra platform-specific options (github, bitbucket etc.) in `dangerfile` script? Like `requester` in gitlab https://github.com/jdalrymple/node-gitlab It would allows me do not fork danger-js only to use it in my case.